apps/mobile/mobile-app/components/ui/gluestack-ui-provider/index.web.tsx

```
@@ -1,95 +1,95 @@
+	let cssVariablesWithMode = ``
+	Object.keys(config).forEach((configKey) => {
+		cssVariablesWithMode +=
+			configKey === 'dark' ? `\n .dark {\n ` : `\n:root {\n`
+		const cssVariables = Object.keys(
+			config[configKey as keyof typeof config],
+		).reduce((acc: string, curr: string) => {
+			acc += `${curr}:${config[configKey as keyof typeof config][curr]}; `
+			return acc
+		}, '')
+		cssVariablesWithMode += `${cssVariables} \n}`
+	})
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)